bin/save_cache

```diff
@@ -37,6 +37,12 @@ if [ -z "$CACHE_BUCKET_NAME" ]; then
 	fi
 fi
 
+# Use with caution – in general it's not a good idea to overwrite a cache entry
+if [ $3 == '--force' ]; then
+	echo "Deleting the existing cache key"
+	aws s3 rm "s3://a8c-ci-cache/$CACHE_KEY"
+fi
```

Comment:
I'm guessing this is here to help devs with debugging or to remove compromised caches?

Reply:
Yep – also the nightly job will regenerate the CocoaPods base caches too, so we use the `--force` flag for that process.